### DISCUSSION TITLE
New ideas

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#199] [Ideas]
- Rank system: By this I mean that every time the player has a certain amount of elo, he earns a rank to make it a ranking type.

- Commands at the end of the game: These commands will be executed from the console to give players coins or etc. every time they win or lose. example:
  Ranked:
     -Winner:
          - give %player% 1 diamond
     -Losser
       - Less %player% 1 diamond

But that is separated by the type of game, example, ranked, unranked, partysplit, partyvsparty, etc...

- New commands:
  -/Kiteditor save (to save the edited kit in the kit editor without the need for signs)
  -/kiteditor leave (To leave the kit editor)
  -/Party ffa or /party split (To enter split or ffa being the leader of a party) can also be accompanied with the subcommand to enter a game with the same example command:
   -/party split boxing
 and take you to a split with the members of the party to the boxing arena.

I hope and some of my ideas will be implemented in the next update, thanks for reading.

**Comment by Stainded** [comment]
I liked this Ideia!

**Reply by liexes** [reply]
Thx

**Comment by liexes** [comment]
@GoodestEnglish

**Reply by GoodestEnglish** [reply]
Not going to add any new features until my exam is done. Will be back in like 2-3 weeks.

**Reply by liexes** [reply]
ok, no problem
